pyproject.toml

```diff
@@ -11,6 +11,7 @@ requires-python = ">=3.10"
 keywords = ["ramalama", "llama", "AI"]
 dependencies = [
   "argcomplete",
+  "PyYaml"
```

Comment:
That's unusual formatting for the dependency name. https://pypi.org/project/PyYAML/ suggests `PyYAML` and https://pyyaml.org/wiki/PyYAMLDocumentation suggests `pyyaml`. I personally prefer the latter.